lib/super_good/solidus_taxjar/api.rb

```diff
@@ -47,6 +47,10 @@ def delete_transaction_for(order)
         taxjar_client.delete_order order.number
       end
 
+      def show_transaction_for(order)
```

Comment:
I am curious if the API here should be to pass the `Spree::Order` record or take the `transaction_id` directly. The reason I am asking that is that only in the case of the initial transaction can we infer the `id` from the order number, for subsequent transactions created for the order (after an update) we won't have the necessary context and may need to just rely on the `transaction_id` to be provided 🤔 

For extra context, me and @benjaminwil  are working on a class to help us generate initial and subsequent transaction id's and we realized we need to have the context of what the `transaction_id` is on TaxJar for an order in order to do that.

Reply:
If we don't store `transaction_id`s in the DB I can't think of a way around this.

We might have to fetch this transaction, then find out if it's been cancelled and fetch the most recent transaction that contains the order number?

Reply:
> We might have to fetch this transaction, then find out if it's been cancelled and fetch the most recent transaction that contains the order number?

I am not sure we can query TaxJar in that way, looking at the documentation for the `show_order` API, it takes a `transaction_id` and gives you back the transaction corresponding to that. So we may have to start with the initial one and keep fetching until we find one that isn't cancelled, but that's exactly what I am aluding to here, we'll inevitably have to fetch a transaction by it `transaction_id` which may be different from the `order.number`.

Reply:
I am okay with that! We may have to change the return type because there could be multiple transactions for a single order, but we can deal with that at a later point. I am fine keeping this consistent with the other methods and handling the actual fetching by the `transaction_id` in the implementation.